marketch.sketchplugin/Contents/Sketch/util.cocoascript

```diff
@@ -261,7 +261,7 @@ var util = {
 
     //NSString to js string
     toJSString: function(NSStr){
-        return new String(encodeURI(NSStr)).toString();
+        return new String(encodeURI(trim(NSStr))).toString();
```

Comment:
@tudou527 ok, just wondering what plugin are you using to print out the logs, will come in handy when  debugging.